src/components/DragDrop/DragDrop.module.css

```diff
@@ -75,16 +75,16 @@
   display: flex;
   flex-shrink: 0;
   flex-direction: column;
-  margin-left: var(--eds-size-2);
+  margin-right: var(--eds-size-2);
```

Comment:
Originally, it was adding margins to the front side of every drag-drop section, and a pseudo-class to chop that margin from the first thing. Flip that around so the margins get added to all but the last section, so that we don't need an override in the most common case.

For future work... this whole component will likely be moved out of EDS with all its core pieces remaining as exported atoms/molecules.